ucm2/Intel/sof-essx8336/HiFi.conf

```diff
@@ -1,71 +1,31 @@
 SectionVerb {
 	EnableSequence [
-		disdevall ""
+		#disdevall ""
 		# Disable all inputs / outputs
 		#   (may be duplicated with disdevall)
-		cset "name='Headphone Switch' off"
-		cset "name='Headset Mic Switch' off"
-		cset "name='Internal Mic Switch' off"
-		cset "name='DAC Mono Mix Switch' off"
+		cset "name='Left Headphone Mixer Left DAC Switch' on"
+		cset "name='Right Headphone Mixer Right DAC Switch' on"
 	]
 }
 
-If.amic {
-	Condition {
-		Type String
-		Empty "${var:DeviceDmic}"
-	}
-	True.SectionDevice."Mic" {
+SectionDevice."Mic" {
 		Comment "Analog Microphone"
 
-		ConflictingDevice [
-			"Headset"
-		]
-
 		EnableSequence [
-			cset "name='Differential Mux' lin1-rin1"
 			cset "name='Internal Mic Switch' on"
+			cset "name='Headset Mic Switch' on"
```

Comment:
Headset mic device was defined bellow (and removed with your change). If you insert a microphone, which Jack control changes? You may use `amixer -c 0 events` command to check the control state changes.